primitives/runtime/Cargo.toml

```diff
@@ -28,6 +28,7 @@ impl-trait-for-tuples = "0.1.3"
 sp-inherents = { version = "2.0.0-rc3", default-features = false, path = "../inherents" }
 parity-util-mem = { version = "0.6.1", default-features = false, features = ["primitive-types"] }
 hash256-std-hasher = { version = "0.15.2", default-features = false }
+either = { version = "1.5", default-features = false }
```

Comment:
does it matter that you did not enable std feature below?

Reply:
Oh I forgot to add it into the std features list below, will update. By default `either` uses std lib, and runtime won't compile if not turning off default features:

> duplicate lang item in crate `std` (which `either` depends on): `panic_impl`.
> duplicate lang item in crate `std` (which `either` depends on): `oom`.